completion/available/rvm.completion.bash

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+# Bash completion support for  RVM.
+# Source: https://rvm.io/workflow/completion
+
+
+_rvm_complete() {
+	[[ -r $rvm_path/scripts/completion ]] && . $rvm_path/scripts/completion
+}
+
+complete -o default -o nospace -F _rvm_complete rvm
```

Comment:
I don't think that this line is necessary. Isn't this already included in the `completion` script here? https://github.com/rvm/rvm/blob/master/scripts/completion#L23